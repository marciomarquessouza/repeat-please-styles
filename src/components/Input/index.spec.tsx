import React from 'react';
import { TextInputProps } from 'react-native';
import { render } from 'react-native-testing-library';
import { PlaceholderInput } from './';

const setup = (props: TextInputProps) => {
	return render(<PlaceholderInput {...props} />);
};

describe('Placeholder Input', () => {
	it('must render the component properly', () => {
		const tree = setup({});
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
