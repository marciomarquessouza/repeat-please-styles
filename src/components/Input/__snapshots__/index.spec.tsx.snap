// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder Input must render the component properly 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
      },
      undefined,
    ]
  }
>
  <TextInput
    allowFontScaling={true}
    placeholderTextColor="#000"
    rejectResponderTermination={true}
    style={
      Array [
        Object {
          "borderBottomColor": "#000",
          "borderLeftColor": "transparent",
          "borderRightColor": "transparent",
          "borderTopColor": "transparent",
          "borderWidth": 1,
          "fontSize": 16,
        },
        undefined,
      ]
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
